pkg/workloads/cortex/lib/util.py

```diff
@@ -116,23 +126,32 @@ def remove_non_empty_directory_paths(paths: List[str]) -> List[str]:
     Then after calling this function, it will look like:
     models/tensorflow/iris/1569001258/saved_model.pb
```

Comment:
Assume that paths are either all absolutes or none of them are. Also error if there are mixed paths (absolutes + relatives).